<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuNet</class>
 <widget class="QDialog" name="MenuNet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTipDuration">
   <number>-5</number>
  </property>
  <widget class="QLabel" name="background_image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>154</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_menu">
    <item>
     <widget class="QCommandLinkButton" name="btn_sys">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="btn_graph">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="btn_bench">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="btn_net">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>311</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_array"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
